fda-model/compute/ImpactAreaScenarioSimulation.cs

```diff
@@ -864,6 +864,7 @@ public SimulationBuilder withStageDamages(List<UncertainPairedData> uncertainPai
                 foreach (UncertainPairedData uncertainPairedData in _sim._damage_category_stage_damage)
                 {
                     _sim.AddSinglePropertyRule(uncertainPairedData.CurveMetaData.DamageCategory + " stage damages", new Rule(() => { uncertainPairedData.Validate(); return !uncertainPairedData.HasErrors; }, uncertainPairedData.GetErrors().ToString()));
+                    _sim.AddSinglePropertyRule("positive-damage", new Rule(() => { return uncertainPairedData.Yvals[uncertainPairedData.Yvals.Length - 1].InverseCDF(0.5) > 0; }, "Stage-damage reflects 0 damage for highest stage", HEC.MVVMFramework.Base.Enumerations.ErrorLevel.Severe));
```

Comment:
This is a property rule. Does that mean there is a property named "positive-damage". Properties should be named like "PositiveDamage".

Reply:
fixed